scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -372,6 +372,46 @@ task oncotate_m2 {
     }
 }
 
+##########
+# Due to a bug in the GATK4 implementation of CollectSequencingArtifactMetrics and the eventual inclusion of picard in GATK4
+#   we need to include picard in this WDL
+task CollectSequencingArtifactMetricsPicard {
+    String entity_id
+    File bam_file
+    String output_location_prepend
+    File picard_jar
+    # /seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.fasta
+    File ref_fasta
+    File ref_fai
+    Int preemptible_attempts
+
+    command {
+        # TODO: Do not hardcode memory requirements
+      java -Xmx4G -jar ${picard_jar} CollectSequencingArtifactMetrics \
+       I=${bam_file} \
+       O=${output_location_prepend} \
+       R=${ref_fasta}
+
+      # Convert to GATK format
+      sed -r "s/picard\.analysis\.artifacts\.SequencingArtifactMetrics\\\$PreAdapterDetailMetrics/org\.broadinstitute\.hellbender\.tools\.picard\.analysis\.artifacts\.SequencingArtifactMetrics\$PreAdapterDetailMetrics/g" "${output_location_prepend}.pre_adapter_detail_metrics" > ${entity_id}.gatk.pre_adapter_detail_metrics
```

Comment:
I broke it up, but maybe not as much as you like.  I do not know how to break up the sed expression.

Done (unless you say otherwise)